execution/inc/leatherman/execution/execution.hpp

```diff
@@ -57,9 +57,10 @@ namespace leatherman { namespace execution {
          */
         preserve_arguments = (1 << 7),
         /**
-         * On Windows, create a new process group for the child process, but not a Job Object.
+         * Create a new process such that it can outlive its parent. This involves running it
+         * in a separate process group on Windows, and in a sparate contract on Solaris.
          */
-        create_new_process_group = (1 << 8),
+        create_detached_process = (1 << 8),
```

Comment:
This is a breaking change, but one that's unused. I'm comfortable including this in 0.10.0 or 1.0.0.